### PR TITLE
adding a feature page for reducing luns from Storage Domain

### PR DESCRIPTION
**Pull request by laravot**
Changes proposed in this pull request:
- Feature page for reducing luns from Storage Domain feature

I confirm that this pull request was submitted according to the [contribution guidelines](/.github/CONTRIBUTING.md): @laravot

This pull request needs review by: @mureinik

### CODE REVIEW

File: source/feature/reduce-luns-from-sd.html.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
```
Comment (rollandf) [inline source/feature/reduce-luns-from-sd.html.md]: The file should be located in 'source/develop/release-management/features/storage'

File: source/feature/reduce-luns-from-sd.html.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+title: Reduce LUNs from a Storage Domain
+category: feature
+authors: laravot
+wiki_category: Feature
+wiki_title: Reduce LUNs from a Storage Domain
+wiki_revision_count: 6
+wiki_last_updated: 2016-09-22
```
Comment (rollandf) [inline source/feature/reduce-luns-from-sd.html.md]: 'wiki_' fields are only for the pages imported from the old wiki.

File: source/feature/reduce-luns-from-sd.html.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+feature_name: Reduce LUNs from Storage Domain
+feature_modules: vdsm, engine
+feature_status: Implementation
```
Comment (rollandf) [inline source/feature/reduce-luns-from-sd.html.md]: Add the version it is planned to be released

File: source/feature/reduce-luns-from-sd.html.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+### Documentation / External references
+
+This feature is using the following LVM commands:
+
+*   [pvmove](https://linux.die.net/man/8/pvmove)
+
+*   [vgreduce](http://linuxcommand.org/man_pages/vgreduce8.html)
+
+### Comments and Discussion
```
Comment (rollandf) [inline source/feature/reduce-luns-from-sd.html.md]: Remove all this section of "Comments and Discussion".
The "Talk" feature is not working on current site.

File: source/feature/reduce-luns-from-sd.html.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+## Benefit to oVirt
+
+Ability to reduce devices from block storage domains.
+
+#### User Experience
+
+1. Refactor the Edit Storage Domain dialog to include option to reduce devices.
+2. Add REST-API support
+
+#### Installation/Upgrade
```
Comment (rollandf) [inline source/feature/reduce-luns-from-sd.html.md]: If there is no impact, remove this section

File: source/feature/reduce-luns-from-sd.html.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+Describe how the feature will effect new installation or existing one.
+
+#### User work-flows
+
+REST-API:
+1. In order to reduce luns the user will submit a POST request to
+/ovirt-engine/api/storagedomains/{storagedomain:id}/reduceluns with the luns it wishes
+to reduce from the storage domain.
```
Comment (rollandf) [inline source/feature/reduce-luns-from-sd.html.md]: Put the URL in tags :
'/ovirt-engine/api/storagedomains/{storagedomain:id}/reduceluns'

File: source/feature/reduce-luns-from-sd.html.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+```xml
+<action>
+<logical_units>
+<logical_unit id="1IET_00010001"/>
+<logical_unit id="1IET_00010002"/>
+</logical_units>
+</action>
+```
+
+2. WEB-UI:
```
Comment (rollandf) [inline source/feature/reduce-luns-from-sd.html.md]: The numbering is not working at expected. Look at the "View" of the feature, it shows "1" again.

**Comment by mureinik** [comment]
@laravot @rollandf ping?

**Comment by rollandf** [comment]
Closing this one and fixing comments in #1029

**Pull request closed without merging**